CONTRIBUTING.md

```diff
@@ -558,6 +558,18 @@ To debug the Azure MCP Server (`azmcp`) when running live tests in VS Code:
 5. Open the Command Palette (`Cmd+Shift+P` on Mac, `Ctrl+Shift+P` on Windows/Linux), select **Debug: Attach to .NET 5+ or .NET Core process**, and enter the `azmcp` process ID
 6. Hit F5 to "Continue" debugging, the debugger should attach to `azmcp` and hit the breakpoint in command file
 
+> [!TIP]
+> **Testing Remote HTTP Mode Locally**
```

Comment:
Testing HTTP mode is covered [in this section](https://github.com/g2vinay/mcp/blob/8ca93726fbd2c5df5576cb242a6e21913c11551c/CONTRIBUTING.md#run-the-azure-mcp-server-in-http-mode). We should include debugging instructions here.